stable/atlantis/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 appVersion: "v0.11.1"
 description: A Helm chart for Atlantis https://www.runatlantis.io
 name: atlantis
-version: 3.11.2
+version: 3.12.2
```

Comment:
when bumping a minor version _(which in this case seems reasonable - it could be argued as a major version bump too)_, then the patch version should be reset to `0` - ie, please change to: `3.12.0`

Reply:
Whoops, sorry about that. Fixed now